### PR TITLE
799 Fix `summarize_num_patients` labels

### LINKED ISSUE #799: Fix label formatting in `summarize_num_patients` and `analyze_num_patients`
### Feature description

Both `summarize_num_patients` and `analyze_num_patients` override the default split labels for `unique` and `nonunique` statistics. This should be changed so that the user is able to use the default split labels or these custom labels (or specify their own labels).

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

### PR DESCRIPTION
**Pull request by edelarua**
Closes #799 

Unblocks https://github.com/insightsengineering/chevron/pull/345

Also updates default format to fixed decimal places in line with other `tern` functions.

### CODE REVIEW

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks a lot! @edelarua

File: R/summarize_num_patients.R [hunk 1]
```diff
@@ -127,8 +122,14 @@ summarize_num_patients <- function(lyt,
                                    var,
                                    .stats = NULL,
                                    .formats = NULL,
-                                   .labels = NULL,
+                                   .labels = c(
+                                     unique = "Number of patients with at least one event",
+                                     nonunique = "Number of events"
+                                   ),
```
Comment (shajoezhu) [inline R/summarize_num_patients.R:128]: @edelarua , do we need a label for ”unique_count“？

File: R/summarize_num_patients.R [hunk 2]
```diff
@@ -167,16 +168,23 @@ summarize_num_patients <- function(lyt,
...
 #' @export
 analyze_num_patients <- function(lyt,
                                  vars,
                                  .stats = NULL,
                                  .formats = NULL,
-                                 .labels = NULL,
+                                 .labels = c(
+                                   unique = "Number of patients with at least one event",
+                                   nonunique = "Number of events"
+                                 ),
```
Comment (shajoezhu) [inline R/summarize_num_patients.R:181]: @edelarua , do we need a label for ”unique_count“？

Comment (edelarua) [reply R/summarize_num_patients.R:181]: @shajoezhu there's no label currently for `unique_count`, but I could add one. Do you have any preference? We could just do `"Number of patients with at least one event (n)"`.

**Pull request merged**